### DISCUSSION TITLE
Self Hosted - Automation History

### DISCUSSION THREAD

**Discussion by shawnweeks** [Budibase/budibase#10528] [Help] [answered]
Did the 1 day of Automation History get removed from the Self-Hosted version of Budibase. After the updating to 2.6.8 I get no automation history, not even for things I just ran.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @shawnweeks,

I've just tested this on version 2.6.8 and the automation history seems to be working fine. Have you published your app and then tried running your automation?

<img width="860" alt="Screenshot 2023-05-10 at 11 45 12" src="https://github.com/Budibase/budibase/assets/126772285/3815c9fc-d3fa-4a15-bead-4af8bbee0a51">

If you are simply testing it within the automation area it won't generate the log.

**Comment by shawnweeks** [comment]
I was looking for the history for a scheduled automation but for some reason it didn't start showing up until I unpublished and then republished the app. Not really sure what was going on.
